server/datastore/mysql/migrations/tables/20240322145615_CreateTableNanoDDMRequests.go

```diff
@@ -0,0 +1,41 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240322145615, Down_20240322145615)
+}
+
+func Up_20240322145615(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE mdm_apple_declarative_requests (
+  id BIGINT NOT NULL AUTO_INCREMENT,
+  created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  enrollment_id VARCHAR(255) NOT NULL,
+  -- Should be one of "tokens", "declaration-items", "status", or "declaration/…/…" where the ellipses reference a declaration on the server
+  message_type VARCHAR(255),
```

Comment:
Should it be `NOT NULL`? We should try to have not nulls as much as possible, to avoid scanning into a non-pointer field and having random panics. But if it has to, it has to of course...